### DISCUSSION TITLE
Elastirini: instruction for searching

### DISCUSSION THREAD

**Discussion by jl63** [castorini/anserini#1742] [Q&A] [answered]
I have several questions for Elastirni:
1. Can anyone provide me with some instructions on how to use Elastirini to do some basic query search like the one in pyserini? Is there any way I can use elastirini with pyserini? 
2. What is the difference between using elastirini only vs. using elastic search only vs. using anserini only? What are the benefits of using elastirini?
I am kind of lost in the documentary and thank you so much for your help!

**Comment by lintool** [comment, marked as answer]
Hi @jl63 -

Apologies for super-late response, but we have a paper exactly answering (2):

<blockquote class="twitter-tweet"><p lang="en" dir="ltr">Forthcoming <a href="https://twitter.com/hashtag/wsdm2022?src=hash&amp;ref_src=twsrc%5Etfw">#wsdm2022</a> demo where <a href="https://twitter.com/joshdevins?ref_src=twsrc%5Etfw">@joshdevins</a> <a href="https://twitter.com/julietibs?ref_src=twsrc%5Etfw">@julietibs</a> and I share thoughts about aligning the research and practice of building search applications. Within the Lucene ecosystem, it is possible to bridge the two worlds - to have your cake and eat it too! <a href="https://t.co/iFyCFL288p">https://t.co/iFyCFL288p</a></p>&mdash; Jimmy Lin (@lintool) <a href="https://twitter.com/lintool/status/1483108162244952066?ref_src=twsrc%5Etfw">January 17, 2022</a></blockquote> 

This is also relevant: https://dl.acm.org/doi/10.1145/3331184.3331401
